backend/infrahub/permissions/local_backend.py

```diff
@@ -54,15 +54,30 @@ def resolve_object_permission(self, permissions: list[ObjectPermission], permiss
 
         return most_specific_permission == PermissionDecision.ALLOW.value
 
+    def resolve_global_permission(self, permissions: list[GlobalPermission], permission_to_check: str) -> bool:
+        if not permission_to_check.startswith("global:"):
+            return False
+
+        _, action, _ = permission_to_check.split(":")
```

Comment:
I think this is fine with regards to this PR. But perhaps we should rethink having `permission_to_check` be a string that we need to split up and apply logic to. Would it be better to use the `Permission` dataclass here. that way we could just compare against `permission_to_check.action` instead?